mkdocs.yaml

```diff
@@ -41,11 +42,23 @@ plugins:
   - mkapi
 nav:
   - About: README.md
-  - Leaderboards: leaderboards
+  - Leaderboards:
+      - leaderboards/README.md
+      - leaderboards/danish.md
+      - leaderboards/dutch.md
+      - leaderboards/english.md
+      - leaderboards/faroese.md
+      - leaderboards/french.md
+      - leaderboards/german.md
+      - leaderboards/icelandic.md
+      - leaderboards/norwegian.md
+      - leaderboards/swedish.md
+      - Compare Models:
+          - leaderboards/radial_plotter.md
```

Comment:
Would this work, if we create an `extras` folder? Naming all the leaderboards manually is sure going to cause bugs in the future. 

```suggestion
      - leaderboards
      - Compare Models:
          - extras/radial_plotter.md
```

Reply:
It works (already discovered missing mixed):
<img width="253" alt="Screenshot 2025-02-04 at 10 34 29" src="https://github.com/user-attachments/assets/c2c22998-3818-43d2-b7b3-40bff76e282f" />

we might consider moving monolingual models into a folder and removing flags
<img width="219" alt="Screenshot 2025-02-04 at 10 38 15" src="https://github.com/user-attachments/assets/ed867439-6b81-4a2c-a046-8ab9e380a4c8" />

Reply:
Would also change Mixed -> Multilingual